app/src/main/res/layout/activity_root.xml

```diff
@@ -2,14 +2,14 @@
 <FrameLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:fitsSystemWindows="true">
```

Comment:
why?

Reply:
because it was a root cause

remove this and run app in release mode :(